packages/document/builder-doc/docs/zh/config/performance/dnsPrefetch.md

```diff
@@ -0,0 +1,23 @@
+- **类型：** `undefined | string | DnsPrefetchOption | Array<DnsPrefetchOption>`
```

Comment:
We can make the type simpler, just support `Array<string | DnsPrefetchOption>` is enough.